cvat-ui/src/components/create-task-page/create-task-page.tsx

```diff
@@ -20,8 +21,16 @@ interface Props {
     installedGit: boolean;
 }
 
-export default function CreateTaskPage(props: Props): JSX.Element {
-    const { error, status, taskId, onCreate, installedGit } = props;
+export default function CreateTaskPage(props: Props & RouteComponentProps): JSX.Element {
+    const {
+        error, status, taskId, location, onCreate, installedGit,
```

Comment:
Let's use ``useLocation`` hook instead of wrapping the component with ``withRouter``